web/src/page_params.ts

```diff
@@ -111,6 +111,7 @@ export const page_params: {
     show_plans: boolean;
     show_webathena: boolean;
     sponsorship_pending: boolean;
+    stop_words?: string[];
```

Comment:
seems like this is only set if `if want("stop_words")`?

Reply:
Yes but we use `always_want` in the web app, so that's always true.